bool inAutonomous
---
int16 error

